.gitignore

```diff
@@ -51,6 +51,7 @@ stamp-h1
 
 # Other
 compile
+build.log
```

Comment:
I think this change can be safely put into other PR and merged earlier :)

Reply:
Yes; will bundle this in a generic PR.